classes/PremiumShopManager.php

```diff
@@ -42,6 +45,30 @@ public function __construct(System $system, User $player) {
         $this->initStatTransferVars();
     }
 
+    ///TEMPORARY SALE LOGIC
+    public function tierThreeSaleActive(): bool {
+        $sale_end = new DateTimeImmutable(self::TIER_THREE_SALE_END);
+        $current_time = new DateTimeImmutable();
+
+        return $current_time < $sale_end;
+    }
+
+    public function saleTimeRemaining(): int {
+        $sale_end = new DateTimeImmutable(self::TIER_THREE_SALE_END);
+        return $sale_end->getTimestamp() - time();
+    }
+
+    public function loadTierThreeSalePrices($current_seal_level): void {
+        if($this->tierThreeSaleActive() && ($current_seal_level == 0 || $current_seal_level == 3)) {
+            $this->costs['forbidden_seal'][3][60] -= 5;
+            $this->costs['forbidden_seal'][3][90] -= 15;
+        }
+        else {
+            $this->costs['forbidden_seal'][3][60] = $this->costs['forbidden_seal_monthly_cost'][3] * 2;
+            $this->costs['forbidden_seal'][3][90] = $this->costs['forbidden_seal_monthly_cost'][3] * 3;
+        }
+    }
```

Comment:
Hmm I think we could probably skip the temporary sale and make these the permanent prices? At least the 3 months discount

Reply:
I think that's reasonable. Should we apply similar discounts to the other two tiers?
Should I leave the 50% refund in place as it is? I think we should give some incentive to switch if someone has 6months of Tier 2

Reply:
Wait, I realized - don't we already have a "apply AK credit from existing seal to a new seal" in place? 
![image](https://github.com/elementum-games/shinobi-chronicles/assets/1691258/97da22f9-6904-4f36-91ed-77aeb487cc1c)

Reply:
Yeah let's leave that refund in 👍

Reply:
For discounts to other tiers - Let's leave it at T3 for now, add the others later when we change AK:USD from a 2:1 to 4:1 ratio